nasa/ow_simulator
# Oceanwater 610 port to noetic

## Linked Issues:
| EPIC ⚡ | [OCEANWATER-577](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-577) |
| :----------- | :----------- |
| Jira Ticket 🎟️ | [OCEANWATER-610](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-610) |
| Github :octocat:  | # |


## Summary of Changes
* Convert scripts from python2 to python3 and update to new noetic MoveIt API.

## Test
* Start the sim and do `roslaunch ow_plexil plexil_node.launch plan:=ReferenceMission1.plx` Make sure the mission completes with no problems.
* Try executing services and actions.
